### PR TITLE
added profile to user model

### PR DESCRIPTION
**Pull request by cimthog**

### CODE REVIEW

File: routes/api/profile.js [hunk 1]
```diff
@@ -57,7 +50,7 @@ router.post('/', async (req, res) => {
 // Retrieve all profiles
 router.get('/', async (req, res) => {
     try {
-        const profiles = await Profile.find();
+        const profiles = await User.find();
```
Comment (olajohn-ajiboye) [inline routes/api/profile.js:53]: Did you test that this works?. I know you usually need to pass an empty object in find to return all.
Like so ```User.find({})```

**Review by olajohn-ajiboye — CHANGES_REQUESTED** [review]

File: routes/api/profile.js (hunk 1 above)
Comment (cimthog) [reply routes/api/profile.js:53]: yes I did test and it's working...  User.find() will return all users in the database

Comment (olajohn-ajiboye) [reply routes/api/profile.js:53]: Cool, all seems good then.

**Review by olajohn-ajiboye — DISMISSED** [review]

File: models/User.js [hunk 2]
```diff
@@ -24,6 +24,8 @@ const UserSchema = new Schema({
         required: true,
         type: String,
     },
+
```
Comment (Way2nnadi) [inline models/User.js:27]: delete extra line

File: package.json [hunk 3]
```diff
@@ -58,6 +58,7 @@
         "enzyme-adapter-react-16": "^1.12.1",
         "express": "^4.16.4",
         "express-session": "^1.16.1",
+        "formidable": "^1.2.1",
```
Comment (Way2nnadi) [inline package.json:61]: where are you using this module?

**Review by Way2nnadi — APPROVED** [review]

File: models/User.js (hunk 2 above)
Comment (cimthog) [reply models/User.js:27]: ok..

File: package.json (hunk 3 above)
Comment (cimthog) [reply package.json:61]: I'm using it to parse form data, for image uploads...shouldn't be on this branch, it's a mistake

**Pull request merged**